examples/stable/foundationdb/blueprint-v2/foundationdb-blueprint.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: foundationdb-blueprint
+actions:
+  backup:
+    kind: CustomResource
+    outputArtifacts:
+      fdbBackup:
+        keyValue:
+          backupPath: "{{ .Phases.TakeBackup.Output.backupPath }}"
+          backupID: "{{ .Phases.TakeBackup.Output.backupID }}"
```

Comment:
I am assuming we are setting this in code, I dont see this being set in the blueprint.

Reply:
That's right, kando sets this internally when Kopia location profile is used

Reply:
ref: https://github.com/kanisterio/kanister/blob/master/pkg/kando/location_push.go#L91